generated/ecs/fields_flat.yml

```diff
@@ -12,6 +12,7 @@
   flat_name: '@timestamp'
   level: core
   name: '@timestamp'
+  order: 0
```

Comment:
Why do we need to keep the order? Shouldn't we just sort alphabetically?

Reply:
We can do either. I was preserving the order because we've been loosely sorting fields in order of importance, and grouping them as well (e.g. the metrics at the end).

Although now that you question this, I realize it makes more sense to sort alphabetically, and explain specifically the relationships between fields. Perhaps we can use the field set description for this.

Reply:
I would try with alphabetical ordering for now to see how well it works. And if we have some fields which need to be on the top (`@timestamp`?) we can still introduce a special param. If we have the order for each entry I worry we easily forget or have duplicates etc. If we introduce a new entry, potentially lots of other entries need update.

Reply:
Now in alphabetical order. Haven't removed the metadata from intermediary files yet (the `order`key), in case we change our mind.